submissions/AaLin-Git/html-css-popup/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Space Hot Tours</title>
+  </head>
+  <body class="body">
+    <div class="wrapper">
+      <header class="header">
+        <nav class="container">
+          <ul class="content">
+            <li>
+              <a class="contentItem" href="#">Mail</a>
+            </li>
+            <li>
+              <a class="contentItem" href="#">Images</a>
+            </li>
+            <li class="hiddenNavbar">
+              <input type="checkbox" id="menuInput" class="input" />
+              <label for="menuInput" class="labelIcon">
+                <img class="menuIcon" src="./img/menu.svg" alt="menu" />
+              </label>
+
+              <section class="menuPopup">
```

Comment:
Check your markup in [validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Faalin-git.github.io%2FHooli-style-Popup%2F)

Reply:
Replaced the sections with the divs